lib/streamlit/cli.py

```diff
@@ -118,17 +120,74 @@ def main_hello():
     _main_run(filename)
 
 
+def _compose_option_parameter(config_option):
```

Comment:
I think a name like this would be clearer: `_convert_config_option_to_click_option`